src/Kawaii.js

```diff
@@ -0,0 +1,52 @@
+import React, { useEffect, useState, useContext } from 'react';
+import {Planet} from 'react-kawaii';
+import {MoodContext} from './MoodContext';
+import {MoodContext2} from './MoodContext2';
+
+
+
+function Kawaii(props) {
+
+    // const [mood, setMood] = useState(true);
+
+    // const toggleMood=() => {
+    //     setMood(prevState => !prevState)
+    // }
+
+    // const moodCond = mood ? 'sad' : 'happy';
+
+    //if loading and loading2 are false, change mood. 
+    const { value, setValue } = useContext(MoodContext)
+    const { value2, setValue2 } = useContext(MoodContext2)
+
+    //how do I say 'if value === 'happy' && value2 === 'happy' {mood = 'happy'}
+    // I can't seem to make it toggle here...so so sad..
+
+    /*I was not able to figure out how to get this thing to switch back to sad when the
```

Comment:
You had almost all of the code written to be able to do this. Since you had the isToggle state attached to the button, the only thing you needed to do was pass down state to the Kawaii in the ArtApi component like this  `<Kawaii isToggled={isToggled}/>` and then add it to the logic on line 43 of the Kawaii component: ` (value === 'blissful' && value2 === 'blissful' && props.isToggled) ?`